pkg/registration/spoke/spokeagent.go

```diff
@@ -246,55 +268,27 @@ func (o *SpokeAgentConfig) RunSpokeAgentWithSpokeInformers(ctx context.Context,
 		// create a ClientCertForHubController for spoke agent bootstrap
 		// the bootstrap informers are supposed to be terminated after completing the bootstrap process.
 		bootstrapInformerFactory := informers.NewSharedInformerFactory(bootstrapKubeClient, 10*time.Minute)
```

Comment:
the comments should be updated. there is no `ClientCertForHubController` now. And please check comments in other places.